TGM/src/main/java/network/warzone/tgm/modules/ChatModule.java

```diff
@@ -62,13 +62,12 @@ private boolean startsWith(String msg) {
     @EventHandler(priority = EventPriority.LOW)
     public void onChat(AsyncPlayerChatEvent event) {
         // Run this code if the chat is currently muted
-        if (!TGM.get().getConfig().getBoolean("chat.enabled")) {
-            if (!event.getPlayer().hasPermission("tgm.togglechat.bypass")) {
-                event.getPlayer().sendMessage(ChatColor.RED + "Chat is currently disabled. If you think this is wrong, please contact a staff member on our Discord.");
-                event.setCancelled(true);
-                return;
-            }
+        if (!TGM.get().getConfig().getBoolean("chat.enabled") && !event.getPlayer().hasPermission("tgm.togglechat.bypass")) {
+            event.getPlayer().sendMessage(ChatColor.RED + "Chat is currently disabled. If you think this is wrong, please contact a staff member on our Discord.");
```

Comment:
I'd change `If you think this is wrong` to `If you believe this is an error`

Reply:
Sure.